quick_sort.py

```diff
@@ -0,0 +1,41 @@
+def quick_sort(data):
+    pivot_list = []
+    small_data = []
+    big_data = []
+    if len(data) <= 1:
```

Comment:
So if this test evaluates to `True`, will you use any of the three symbols you defined above it?  If not, why define them first?